src/pages/iou/MoneyRequestSelectorPage.js

```diff
@@ -0,0 +1,147 @@
+import {withOnyx} from 'react-native-onyx';
+import {View} from 'react-native';
+import React, {useRef} from 'react';
+import lodashGet from 'lodash/get';
+import PropTypes from 'prop-types';
+import {createMaterialTopTabNavigator} from '@react-navigation/material-top-tabs';
+import ONYXKEYS from '../../ONYXKEYS';
+import FullPageNotFoundView from '../../components/BlockingViews/FullPageNotFoundView';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithBackButton from '../../components/HeaderWithBackButton';
+import TabSelector from '../../components/TabSelector/TabSelector';
+import CONST from '../../CONST';
+import useLocalize from '../../hooks/useLocalize';
+import * as IOUUtils from '../../libs/IOUUtils';
+import Navigation from '../../libs/Navigation/Navigation';
+import styles from '../../styles/styles';
+import MoneyRequestAmount from './steps/MoneyRequestAmount';
+import ReceiptSelector from './ReceiptSelector';
+import * as IOU from '../../libs/actions/IOU';
+import DragAndDropProvider from '../../components/DragAndDrop/Provider';
+import OnyxTabNavigator from '../../libs/Navigation/OnyxTabNavigator';
+import Permissions from '../../libs/Permissions';
+
+const TopTab = createMaterialTopTabNavigator();
```

Comment:
Rather than creating another navigator here, we should export this as a named export from `OnyxTabNavigator` then just import it here like:

```js
import OnyxTabNavigator, {TopTab} from '../../libs/Navigation/OnyxTabNavigator';
```